src/components/Goals/BikeGoal/helpers.spec.js

```diff
@@ -13,10 +13,10 @@ jest.mock('cozy-flags', () => name => {
 
 describe('isGoalCompleted', () => {
   it('should return true', () => {
-    const timeseries = [
-      { startDate: '2021-01-01T00:00:00', endDate: '2021-01-02T00:00:00' },
-      { startDate: '2021-03-01T00:00:00', endDate: '2021-03-02T00:00:00' }
-    ]
+    const timeseries = Array.from({ length: 30 }, (_, index) => ({
+      startDate: new Date(2021, 1, index + 1),
+      endDate: new Date(2021, 1, index + 1)
```

Comment:
(does not really matter here, but for your information, reminder that months start at 0 in JS)